database/intra_H_migration/TS_groups.py

```diff
@@ -2,8 +2,8 @@
 # encoding: utf-8
 
 name = "intra_H_migration/TS_groups"
-shortDesc = u""
-longDesc = u"""
+short_desc = u""
```

Comment:
Did they change the syntax of RMG-database??

Reply:
They have not, but we changed the syntax of ours. Should we change it back?